packages/service/src/db.test.ts

```diff
@@ -9,14 +10,38 @@ test("crdts can be persisted", () => {
   const state = Y.encodeStateAsUpdate(ydoc);
 
   // persist this change to the database
-  // ????
+  const doc = await db
+    .insert("document", {
+      value: ytext.toJSON(),
+      origin: Buffer.from(state),
+      web_rtc_key: "",
+    })
+    .run(pool);
+
+  // const x: schema.document.Insertable = {
+  //   value: ytext.toJSON(),
+  //   origin: Buffer.from(state),
+  //   web_rtc_key: "",
+  // };
+  // const doc = await db.sql<schema.document.SQL, schema.document.JSONSelectable>`INSERT INTO ${"document"} (${db.cols(x)}) VALUES (${db.vals(x)})`.run(
+  //   pool
+  // );
+  // console.log(doc);
 
   // get the change back from the database
-  const stateFromDatabase = state;
+  const stateFromDatabase = await db
+    .select("document", {
+      id: doc.id,
+    })
+    .run(pool);
+
+  // Question: What is the difference between the shortcut select query and the SQL select query? SQL query does not work.
+  // const stateFromDatabase = await db.sql<schema.document.SQL, schema.document.JSONSelectable[]>`SELECT * FROM ${"document"} WHERE ${{id: doc.id}}`.run(pool);
```

Comment:
I don't think `${{id: doc.id}}` makes sense here, otherwise it looks good. I think what you want is

```js
`SELECT * FROM ${"document"} WHERE id = ${db.value(id)}`
```